pyproject.toml

```diff
@@ -29,6 +29,7 @@ eth-account = "<0.8.0"
 eth-keys = "<0.4.0"
 autoflake = "^2.0.0"
 pycryptodome = "^3.16.0"
+rlp = "<3"
```

Comment:
seems as if i need to commit an updated lock file

the output of the error is saying to run

```
poetry lock --no-update
```

Is it as simple as that and then pushing the updated lock file?

Reply:
no, it's easier 😇

actually when you want to add a lib, you don't (never) update the `pyproject.toml` by hand. Instead, you want `poetry` (or the other package manager you use) to make it for you as it'll check dependencies versions consistency.

So you should use instead

```
poetry add "rlp<3" --group dev
```

and it'll be update the `pyproject.toml`, the `poetry.lock` file and install the lib in your local interpreter.

Reply:
thank you!